README.md

```diff
@@ -69,8 +69,8 @@ const ip = Email.parse('info@[127.0.0.1]');
 const isIP = ip.isIpBased; // true
 ```
 
-### InternationBankAccountNumber
-Represnts a IBAN.
+### International Bank Account Number
+Represents an IBAN.
```

Comment:
Shouldn't the type on line 76 be "InternationalBankAccountNumber". I changed the title.

Reply:
Ah now I see... Slow...